### PR TITLE
Fix pr warnings

### PR DESCRIPTION
**Pull request by Emmanueldmlr**
fixes https://github.com/EveripediaNetwork/issues/issues/111

### CODE REVIEW

File: src/components/Elements/ImageInput/ImageInput.tsx [hunk 1]
```diff
@@ -43,7 +44,7 @@ const ImageInput = ({
           alignItems="center"
           gap={4}
         >
-          <img src={imgSrc} alt="input" />
+          <Image src={imgSrc} alt="input" />
```
Comment (JustinCast) [inline src/components/Elements/ImageInput/ImageInput.tsx:47]: @Emmanueldmlr make sure the image being pasted works with Image component from nextjs

Comment (Emmanueldmlr) [reply src/components/Elements/ImageInput/ImageInput.tsx:47]: Yes. You get to view the image after upload right?

Comment (JustinCast) [reply src/components/Elements/ImageInput/ImageInput.tsx:47]: Actually I can't see the preview to check if it's working

**Comment by Emmanueldmlr** [comment]
@JustinCast  @kesar , kindly review

**Review by kesar — APPROVED** [review]

**Comment by JustinCast** [comment]
@Emmanueldmlr the image input seems to not be working anymore

**Comment by JustinCast** [comment]
@Emmanueldmlr for instance, in this link the image input is working as expected https://monopedia-ui-git-close-109-prediqt.vercel.app/create-wiki

File: src/components/Elements/ImageInput/ImageInput.tsx (hunk 1 above)
Comment (JustinCast) [reply src/components/Elements/ImageInput/ImageInput.tsx:47]: @Emmanueldmlr

Comment (kesar) [reply src/components/Elements/ImageInput/ImageInput.tsx:47]: opened new issue 👍🏻  https://github.com/EveripediaNetwork/issues/issues/130

**Pull request merged**